src/SDKs/SqlManagement/Sql.Tests/ManagedDatabaseRestoreScenarioTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Azure.Management.Network.Models;
+using Sku = Microsoft.Azure.Management.Sql.Models.Sku;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+
+namespace Sql.Tests
+{
+    // More tests will be added here once we start working on RestorableDroppedDatabase clients
+    //
+    public class ManagedDatabaseRestoreScenarioTests
+    {
+        [Fact]
+        public void ShortTermRetentionOnLiveDatabase()
+        {
+            SqlManagementTestContext Context = new SqlManagementTestContext(this);
+            ResourceGroup resourceGroup;
+            ManagedInstance managedInstance;
+            string testPrefix = "manageddatabaserestorescenariotest-";
+
+            try
+            {
+                SqlManagementClient sqlClient = Context.GetClient<SqlManagementClient>();
```

Comment:
nit: use `using` block instead of try-finally to dispose

Reply:
Will do